exoscale/resource_exoscale_elastic_ip.go

```diff
@@ -279,6 +293,15 @@ func resourceElasticIPUpdate(ctx context.Context, d *schema.ResourceData, meta i
 
 	var updated bool
 
+	if d.HasChange(resElasticIPAttrLabels) {
+		labels := make(map[string]string)
+		for k, v := range d.Get(resElasticIPAttrLabels).(map[string]interface{}) {
+			labels[k] = v.(string)
+		}
+		elasticIP.Labels = &labels
```

Comment:
I don't see the need to copy labels by value. Assuming my previous suggestion works (casting to `map[string]string`), we can just set pointer to that. I could be wrong though, to be tested.

Reply:
Same snippet is all over our codebase, but if simplification works we might as well start code cleanup from this.